content/blog/2021/10/05/OptimizeYourPython.adoc

```diff
@@ -0,0 +1,438 @@
+= Optimize your Python
```

Comment:
This title implies that we're optimizing Python code.
Instead, we're optimizing planning problem with Python.

Also, it would be good to have keywords in this title, like "AI" and "constraint solver/solving".

A) How about "A new AI constraint solver for Python: OptaPy"
B) "Want to use OptaPlanner in Python? Now you can!" Nah, I wouldn't target the messaging to "OptaPlanner in Python". We want to attrackt new Python users, not migrate the existing Java users (we are still Java first).
C) "A new Python constraint solver is born: OptaPy" (Burr rule: most important keyword first)

-1 for B. I kinda prefer A), but C) is good too. Other proposals welcome :)